### PR TITLE
doc: Add examples to readmes, and update stale docs

### PR DESCRIPTION
**Pull request by shimonp21**

### CODE REVIEW

File: plugins/source/heroku/client/spec.go [hunk 1]
```diff
@@ -13,6 +13,8 @@ import (
 type Spec struct {
 	Token string `yaml:"token,omitempty" json:"token"`
 
+	// These don't actually work (throw a "json unknown field" error) - but not a high-piority to fix this,
```
Comment (hermanschaaf) [inline plugins/source/heroku/client/spec.go:16]: I think it'll work if we change `yaml` to `json` in all the cases here. But yeah, not a high priority.

Comment (hermanschaaf) [reply plugins/source/heroku/client/spec.go:16]: In fact, maybe we should just remove these options until further notice?

**Review by hermanschaaf — APPROVED** [review]
Very nice! ✨

File: plugins/source/heroku/client/spec.go (hunk 1 above)
Comment (shimonp21) [reply plugins/source/heroku/client/spec.go:16]: Will do in a follow up PR

Comment (hermanschaaf) [reply plugins/source/heroku/client/spec.go:16]: Yup, not a blocker for this PR

**Pull request merged**